admin/admin-ui/client/source/layout/MainLayout/Header/HeaderContent/Profile/index.tsx

```diff
@@ -128,11 +128,11 @@ const Profile = () => {
                                             <Grid container justifyContent="space-between" alignItems="center">
                                                 <Grid item>
                                                     <Stack direction="row" spacing={1.25} alignItems="center">
-                                                        <Avatar alt="profile user" src='/images/avatar-1.png' sx={{ width: 32, height: 32 }} />
+                                                        <Avatar alt="profile user" sx={{ width: 32, height: 32 }} />
                                                         <Stack>
                                                             <Typography variant="h6">{user.user}</Typography>
                                                             <Typography variant="body2" color="textSecondary">
-                                                                UI/UX Designer
+                                                                Org1
```

Comment:
Is this a hardcoded value?

Reply:
Here we should add the relevant organization. For that, there is a to-do implementation from the rest API side. So added a comment with this 20fa35609936424d278da31ea6c0b3d60d88b5ed